crates/swc_ecma_minifier/src/compress/optimize/sequences.rs

```diff
@@ -1315,20 +1315,23 @@ where
         #[cfg(feature = "debug")]
         let _tracing = {
             let b_str = dump(&*b, false);
-            let a_id = a.id();
+            let a = match a {
+                Mergable::Expr(e) => dump(*e, false),
+                Mergable::Var(e) => dump(*e, false),
+            };
 
             Some(
                 span!(
                     Level::ERROR,
                     "merge_sequential_expr",
-                    a_id = tracing::field::debug(&a_id),
+                    a = tracing::field::debug(&a),
                     b = &*b_str
                 )
                 .entered(),
             )
         };
 
-        if b.is_class() || b.is_fn_expr() || b.is_arrow() {
+        if b.is_lit() || b.is_class() || b.is_fn_expr() || b.is_arrow() {
```

Comment:
I'm confused by this change. `is_lit` seems like a shorthand for `is_literal`, and wouldn't we be able to optimize a literal value?